Lib/test/test_readline.py

```diff
@@ -210,8 +210,10 @@ def display(substitution, matches, longest_match_length):
         self.assertIn(b"result " + expected + b"\r\n", output)
         self.assertIn(b"history " + expected + b"\r\n", output)
 
-    @unittest.skipIf(is_editline,
```

Comment:
Right, we have several reasons for skiping:
- libedit on os x 10.11.6 does not implement history size - I tested using the libedit special format.
- newer version on os x not tested yet, I don't want to add a possibly broken test
- readline < 6.0 does not support history size